packages/calcite-components/src/components/action-group/action-group.tsx

```diff
@@ -207,10 +212,10 @@ export class ActionGroup
 
   render(): VNode {
     return (
-      <Fragment>
+      <div aria-label={this.label} class={CSS.container}>
```

Comment:
Does this behave the same way as setting `aria-label` directly on the host?

Reply:
@geospatialem can you test for me?

I could just add it on the host but I'd rather not modify the host if not necessary.

Reply:
`aria-label` context isn't provided to NVDA, but if we can add a `role="group"` to the `div`, both JAWS and NVDA provide context in FF and Chrome.

Also, wondering with this ✨ fantastic ✨ update, if we could add a `label` into the demos as an example. Maybe [this one](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/demos/action.html#L318)? WDYT?

Reply:
Updated! 🪄